adoption.md

```diff
@@ -0,0 +1,33 @@
+# Adopt DataJoint Elements 
+You have several options for adopting DataJoint workflows for your own experiments. 
+
+## Support from DataJoint NEURO
+Our team provides support to labs to adopt DataJoint workflows in their research. 
+
+This includes:
+ * User training 
+ * Developer training  
+ * Data and computation hosting 
+    * on your premises 
+    * using your own cloud accounts
+    * fully managed cloud hosting by DataJoint 
+ * Workflow execution  
+    * configuration and automation
+    * optional fully managed service by DataJoint Neuro
+ * Interfaces for data entry, export and publishing 
+
+During alpha and beta testing phases, these services may be subsidized by the grant funding for qualified research groups. 
+
+To select the support service, [apply here](https://elements.datajoint.io/apply)
```

Comment:
We can just embed here the application to be an alpha tester/validation site. Have you guys determined what we need to collect when submitting?

Reply:
* Contact name and email
*  Institution and lab
*  How will you set up the MySQL server:
   (a) we will configure it ourselves
   (b) please set up a cloud server for our project for the testing period (10 sessions)